bitcoin_client/tests/test_sign_psbt_legacyapp.py

```diff
@@ -97,12 +101,13 @@ def test_sign_psbt_singlesig_wpkh_1to2(client: Client):
     #   "pubkey" : "03ee2c3d98eb1f93c0a1aa8e5a4009b70eb7b44ead15f1666f136b012ad58d3068",
     #   "signature" : "3045022100ab44f34dd7e87c9054591297a101e8500a0641d1d591878d0d23cf8096fa79e802205d12d1062d925e27b57bdcf994ecf332ad0a8e67b8fe407bab2101255da632aa01"
 
-    assert len(result) == 1
-
-    idx, _, sig = result[0]
-    assert idx == 0
-    assert sig == bytes.fromhex(
-        "3045022100ab44f34dd7e87c9054591297a101e8500a0641d1d591878d0d23cf8096fa79e802205d12d1062d925e27b57bdcf994ecf332ad0a8e67b8fe407bab2101255da632aa01")
+    assert result == [(
+        0,
+        bytes.fromhex("03ee2c3d98eb1f93c0a1aa8e5a4009b70eb7b44ead15f1666f136b012ad58d3068"),
```

Comment:
You could use variables to store these values. This will improve readability and you can remove the comments above :)